tests/TestHarness/Cluster.py

```diff
@@ -705,7 +712,9 @@ def createAccountKeys(count):
 
     # create account keys and import into wallet. Wallet initialization will be user responsibility
     # also imports defproducera and defproducerb accounts
-    def populateWallet(self, accountsCount, wallet):
+    def populateWallet(self, accountsCount, wallet, accountNames: list=None, createProducerAccounts: bool=True):
+        if accountsCount == 0 and len(accountNames) == 0:
+            return True
```

Comment:
When `accountNames` is `None`, this line would cause `TypeError: object of type 'NoneType' has no len()`

Reply:
Addressed: 500814876e1e